src/components/expansion/ExpansionRuleForm.svelte

```diff
@@ -94,6 +104,8 @@
           authoring_command_dict_id: ruleDictionaryId,
           authoring_mission_model_id: ruleModelId,
           expansion_logic: ruleLogic,
+          name: ruleName,
+          ...(ruleDescription && { description: ruleDescription }),
```

Comment:
Not sure why, but I'm not able to update an expansion rule after creating it. This error is originating in the `updateExpansionRule` function:

```
Error: unexpected null value for type 'String'
```

Reply:
Taking a look

Reply:
Not able to repro yet. Is there a more specific scenario here? Also just pushed some potentially related fixes (though still couldn't repro pre-fixes).

Reply:
Think I ran into the scenario you were talking about, had to do with the expansion rule name not being factored into the disabling of the save button.

Reply:
Should be resolved now.